### ISSUE TITLE
Cannot update to Blosc 0.2.0 on Windows

### ISSUE THREAD

**Issue by BoundaryValueProblems** [stevengj/Blosc.jl#37]
Blosc was working on Windows machines until recently, but when I ran ```Pkg.update()```, it failed with the following errors:
```julia
INFO: Building Blosc
Exception calling "DownloadFile" with "2" argument(s): "An exception occurred during a WebClient request."
At line:1 char:1
+ (new-object net.webclient).DownloadFile("https://bintray.com/artifact ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException

====================================================[ ERROR: Blosc ]====================================================

LoadError: failed process: Process(`'C:\Windows\System32\WindowsPowerShell\v1.0\powershell' -NoProfile -Command '(new-object net.webclient).DownloadFile("https://bintray.com/artifact/download/julialang/generic/libblosc64-1.11.1.dll", "libblosc.dll")'`, ProcessExited(1)) [1]
while loading C:\Users\xxx\.julia\v0.5\Blosc\deps\build.jl, in expression starting on line 11

========================================================================================================================

====================================================[ BUILD ERRORS ]====================================================

WARNING: Blosc had build errors.

 - packages with build errors remain installed in C:\Users\xxx\.julia\v0.5
 - build the package(s) and all dependencies with `Pkg.build("Blosc")`
 - build a single package by running its `deps/build.jl` script

========================================================================================================================
```
I didn't have any problems updating Blosc on my Mac OS X and Ubuntu/Linux machines. So, this is certainly a Windows-specific problem.
Thanks a lot for your help!
BVPs

**Comment by stevengj** [comment]
Looks like a network error during download?  Can you download `https://bintray.com/artifact/download/julialang/generic/libblosc64-1.11.1.dll` manually?

**Comment by BoundaryValueProblems** [comment]
Yes, I can manually download that file without any problem. And, I still cannot build Blosc; the same error still occurs. So, I don't think this is a network problem.

**Comment by BoundaryValueProblems** [comment]
Perhaps, would the following stackoverflow post be helpful?
http://stackoverflow.com/questions/13712462/error-using-client-downloadfile-in-powershell-script

**Comment by stevengj** [comment]
Possibly related to JuliaPackaging/WinRPM.jl#40?

**Comment by BoundaryValueProblems** [comment]
@stevengj, somehow, I could build Blosc just now. It's quite strange. I didn't change any firewall setting. But I just ran Pkg.update(), which updated many packages including Compat, SHA, and Atom, and more over I updated two Juno-Atom packages (atom-ink and atom-julia-client), and that forced me to restart atom. Then, Pkg.build("Blosc") worked without any error... I'm puzzled if those other package updates affected my building the Blosc package...
